app/services/subcases-service.js

```diff
@@ -1,11 +1,51 @@
 import Service, { inject } from '@ember/service';
 import { ajax } from 'fe-redpencil/utils/ajax';
+import { trimText } from 'fe-redpencil/utils/trim-util';
 import moment from 'moment';
 
 export default Service.extend({
   store: inject(),
   intl: inject(),
 
+  /**
+   * Create a subcase for a publicationFlow
+   *
+   * @param _case
+   * @param _publicationFlow
+   * @param subcaseTypeObject
+   * @param shortTitle
+   * @param title
+   * @returns {Promise<any>}
+   */
+  async createSubcaseForPublicationFlow(_case, _publicationFlow, subcaseTypeObject, shortTitle, title) {
+    const creationDatetime = moment().utc()
+      .toDate();
+
+    // create Subcase.
+    const subcase = await this.store.createRecord('subcase', {
+      type: subcaseTypeObject,
+      shortTitle: trimText(shortTitle),
+      title: trimText(title),
+      // TODO: wat moeten we hier in steken?
+      // confidential: confidential || false,
+      // showAsRemark: showAsRemark || false,
+      case: _case,
```

Comment:
Deze zou in principe weg mogen? Deze procedurestap maakt deel uit van de publicatie(aangelegenheid) en niet van de besluitvormingsaangelegenheid(=bij ons dossier).